src/d3a/models/strategy/pv.py

```diff
@@ -221,14 +221,15 @@ def gaussian_energy_forecast_kWh(self, time_in_minutes=0):
             gauss_forecast = 0
 
         else:
-            gauss_forecast = self.max_panel_power_W * math.exp(
+            gauss_forecast = self.capacity_kW * math.exp(
                 # time/5 is needed because we only have one data set per 5 minutes
 
                 (- (((round(time_in_minutes / 5, 0)) - 147.2)
                     / 38.60) ** 2
                  )
             )
-        return round(convert_W_to_kWh(gauss_forecast, self.area.config.slot_length), 4)
+
+        return round(convert_kW_to_kWh(gauss_forecast, self.area.config.slot_length), 4)
```

Comment:
I think we're usually rounding to a shared precision:
```suggestion
from d3a.constants import DEFAULT_PRECISION
        # ...
        return round(convert_kW_to_kWh(gauss_forecast, self.area.config.slot_length), DEFAULT_PRECISION)
```

Reply:
AFAIR, It made some integrationtests fail which where not that easy to adapt. That is why i left this for a later change.